src/main/java/com/alkemy/ong/model/request/CreateMemberRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.alkemy.ong.model.request;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateMemberRequest {
+
+  @NotBlank(message = "Name is mandatory")
+  @Pattern(regexp = "^[A-Za-z]*$", message = "The name has to contain only letters")
+  private String name;
+  private String facebookUrl;
+  private String instagramUrl;
+  private String linkedinUrl;
+  @NotBlank(message = "Image is mandatory")
```

Comment:
MINOR: para ser consistente con los mensajes que existen en el proyecto lo modificaría a: "Image cannot be null or empty."